android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/annotation/core/pdf/DocumentationOpener.kt

```diff
@@ -0,0 +1,5 @@
+package com.github.barteksc.pdfviewer.annotation.core.pdf
+
+interface DocumentationOpener {
```

Comment:
Use a fun interface here - https://kotlinlang.org/docs/fun-interfaces.html#sam-conversions

Reply:
Also instead of DocumentationOpener call this `onAnnotatitionPressListener` to be consistent with other listener interfaces

Reply:
Done and moved it to the folder with the other listeners